hilti/src/ast/scope-lookup.cc

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2020 by the Zeek Project. See LICENSE for details.
+
+#include <utility>
+
+#include <hilti/ast/ctors/enum.h>
+#include <hilti/ast/declarations/constant.h>
+#include <hilti/ast/declarations/module.h>
+#include <hilti/ast/scope-lookup.h>
+#include <hilti/base/logger.h>
+
+using namespace hilti;
```

Comment:
This is the style we use in many other places, but an alternative and slightly less ambiguous style would be to move all these definitions into an explicit namespace instead of relying on the compiler to figure out names matching declarations. Maybe something we could do since this is a new file.
```.cpp
namespace hilti {

namespace logging::debug { ... }

std::pair<bool, Result<std::pair<NodeRef, ID>>> scope::detail::lookupID(...

} // namespace hilti
```

Reply:
similar argument here: we can discuss changing style, but I prefer consistency over changing just individual files, so leaving as is.